internal/backends/backend.go

```diff
@@ -79,6 +86,107 @@ func (bc *backendContract) Close() {
 	resource.Untrack(bc, bc.token)
 }
 
+// CreateUser stores a new user in the given database and Backend, ignoring error if the user already exists.
```

Comment:
:warning: This is called from msg_createuser.go, it will introduce incompatibility with MongoDB if it ignores error if user exists.